### ISSUE TITLE
Alternative to this extension (due to deprecation?)

### ISSUE THREAD

**Issue by Styxxy** [microsoft/azdo-databricks#50]
Since this extension seems to be marked as deprecated (still open PR #43 ),

Is there any alternative to the tasks provided here?

**Comment by bjtho08** [comment]
@riserrad Is this project abandoned? If not, who should we contact about issues?

**Comment by Styxxy** [comment]
Any update or information regarding this question?

**Comment by riserrad** [comment]
This project is on path for deprecation, unless anyone interested in taking over shows up.

**Comment by Styxxy** [comment]
@riserrad That we noticed. Do you have a suggestion for alternative? Or are we on our own?

**Comment by bjtho08** [comment]
@riserrad do you still have write access to this repository or is it only the Microsoft bot account at this point? I might consider taking over, if I find the time to learn basic ts, but it will be at least a few months in the future. But if another set of tasks exists, that can fill the gap from this repository, I don't see a point in trying to save it.

**Comment by riserrad** [comment]
@bjtho08 yes I still have access to this repo. I am not aware of any set of Tasks that are similar to this one though.

Let me know if I can help you by granting access, etc.

**Comment by Styxxy** [comment]
@riserrad Any clarification on this issue yet?
If still deprecated, do you know an alternative?

**Comment by Styxxy** [comment]
@riserrad Any update on this repo? Consider fully deprecating this task.
Are there any alternatives, or should we move to bash/cli tasks and go back to scripting ourselves?
